packages/back-end/src/models/SavedGroupModel.ts

```diff
@@ -144,14 +140,22 @@ export async function deleteSavedGroupById(id: string, organization: string) {
 export function toSavedGroupApiInterface(
   savedGroup: SavedGroupInterface
 ): ApiSavedGroup {
+  const values =
+    savedGroup.source === "inline"
+      ? getLegacySavedGroupValues(savedGroup.condition, savedGroup.attributeKey)
+      : [];
+
+  // Populate the `values` field from legacy saved groups with a really simple condition
```

Comment:
Nitpick: I think this comment (or a duplicate of it) would be useful to have near the util function definition